### PR TITLE
Fix Tutorial tab is still visible even if the tutorial.html file has been deleted.

### PR DESCRIPTION
**Pull request by edyedy123**
Hey,
I noticed that this bug has been open for over a month so I decided to fix it.

I made the fix on dist/bramble.js, which is compressed or something for performance reasons i'm guessing so it may be hard to see the change i made so here is a picture of the uncompressed version:
image
fixes thimble bug #1279 and thimble bug #1596
https://github.com/mozilla/thimble.mozilla.org/issues/1279
https://github.com/mozilla/thimble.mozilla.org/issues/1596

### CODE REVIEW

**Comment by humphd** [comment]
@edyedy123 thanks for looking into this.  I had thought @MargarytaChepiga was working on this, but maybe I'm wrong and she doesn't want to do it.  

At any rate, you need to make the changes you've done in `src/bramble/client/main.js` instead of `dist/bramble.js`.  The latter is the built version, and will get overwritten with each call to `npm run build`.  Can you please do a new commit to fix that?

File: src/bramble/client/main.js [hunk 1]
```diff
@@ -750,7 +750,14 @@ define([
                         // If the path was a file, immediately unlink
                         // trigger the event, and call the callback
                         if(err.code === "ENOTDIR") {
-                            return shell.rm(path, genericFileEventFn("fileDelete", path, callback));
+                            return shell.rm(path, genericFileEventFn("fileDelete", path, function(err) {
+                                wrappedCallback = callback;
```
Comment (Pomax) [inline src/bramble/client/main.js]: this is missing a `var` - without it, this variable will be globally scoped, not function scoped, and very unpredictable things will happen if this triggers twice.

**Comment by MargarytaChepiga** [comment]
@edyedy123 @humphd  I have already done a pull request for this issue but I accidentally submitted it to Adobe instead of Mozilla.

Here's a a link to my pull request:
https://github.com/adobe/brackets/pull/13121

I will resubmit the pull request soon. Apologies for the confusion.

**Comment by MargarytaChepiga** [comment]
Here is the PR for this fix: https://github.com/mozilla/brackets/pull/693

**Comment by humphd** [comment]
@edyedy123 thanks for fixing this, sorry for the confusion.  If you want to still include it in your releases, that's fine.  I've merged the other PR https://github.com/mozilla/brackets/pull/693.

**Pull request closed without merging**